jxl_macros/src/lib.rs

```diff
@@ -5,13 +5,16 @@
 
 extern crate proc_macro;
 
+#[cfg(feature = "test")]
 use std::{fs, path::Path};
 
 use proc_macro::TokenStream;
 use proc_macro_error2::{abort, proc_macro_error};
 use proc_macro2::TokenStream as TokenStream2;
 use quote::quote;
-use syn::{DeriveInput, Ident, Meta, parse_macro_input};
+#[cfg(feature = "test")]
+use syn::Ident;
```

Comment:
Should we move these use declarations at function level?

Reply:
Done